superlesson/storage/store.py

```diff
@@ -28,15 +28,37 @@ def _get_storage_path(self, filename: str, format: Format) -> Path:
             return self._storage_root / f"{filename}.json"
         return self._lesson_root / f"{filename}.txt"
 
-    @staticmethod
-    def _parse_txt(txt_path: Path) -> list[str]:
-        return [
-            format_transcription(text)
-            for text in re.split(r"====== SLIDE .* ======", txt_path.read_text())[1:]
-        ]
-
-    @staticmethod
-    def _parse_json(json_path: Path) -> list[dict[str, Any]]:
+    def _parse_txt(self, txt_path: Path) -> dict[str, Any]:
+            split_texts = re.split(r"====== SLIDE (\S*) \((.*)\) ======", txt_path.read_text())[1:]
+
+            transcriptions = split_texts[2::3]
+
+            timeframes = [
+                {"start": start_end.split(" - ")[0], "end": start_end.split(" - ")[1]}
+                for start_end in split_texts[1::3]
```

Comment:
you don't need to split twice here:

```suggestion
                {"start": start, "end": end}
                for start, end in map(lambda t: t.split(" - "), split_texts[1::3])
```